presto-docs/src/main/sphinx/language/time.rst

```diff
@@ -0,0 +1,57 @@
+========================
+Legacy And New Timestamp
+========================
+
+New ``TIMESTAMP`` and ``TIME`` semantics align the types with the SQL standard.
+See the following sections for details.
+
+.. note::
+
+   The new ``TIMESTAMP`` semantics is still experimental. It's recommended to keep
+   the legacy ``TIMESTAMP`` semantics enabled. You can experiment with the new semantics
+   by enabling in the global configuration or session level. The legacy semantics
+   may be deprecated in a future release.
+
+Enabling/Disabling
+------------------
+
+The legacy ``TIMESTAMP`` semantics can be set to ``true`` or ``false``
+by setting ``deprecated.legacy-timestamp`` in the ``config.properties.``
+
+Additionally, there is a session level property ``legacy_timestamp``
+that can be set.
+
+TIMESTAMP semantic changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Previously, the ``TIMESTAMP`` type described an instance in time in the Presto session's time zone.
+Now, Presto treats ``TIMESTAMP`` values as a set of the following fields representing wall time:
+
+ * ``YEAR OF ERA``
```

Comment:
Could we simplify these to year, month, day, hour, minute, second?

I'm not sure that "of era", "of year", etc., add any value (it seems obvious what they mean, and I can't think of any other reasonable alternative interpretation)

Reply:
I think these terms follow terminology that can be found in Java Time and Joda Time. 

- `java.time.LocalDateTime#getDayOfYear`
- `java.time.LocalDateTime#getDayOfMonth`
- `java.time.LocalDateTime#getDayOfWeek`

- `org.joda.time.LocalDateTime#getDayOfYear`
- `org.joda.time.LocalDateTime#getDayOfMonth`
- `org.joda.time.LocalDateTime#getDayOfWeek`

Joda LDT has 3 methods `getYear*`, but Java Time's LDT has just one `getYear`.

I can remove these `OF ..` from here. Please decide.